src/main/java/com/itschool/eventmanagment/services/EventServiceImp.java

```diff
@@ -36,13 +36,19 @@ public EventDTO createEvent(EventDTO eventDTO) {
     }
 
     @Override
-    public List<EventDTO> deleteEvent() {
-        return null;
+    public void  deleteEvent(Long id) {
+    Event event = eventRepository.findById(id).orElseThrow(()-> new EventNotFoundException("Event not found with id: " + id ));
+    eventRepository.deleteById(id);
     }
 
     @Override
-    public List<EventDTO> updateEvent(EventDTO eventDTO) {
-        return null;
+    public EventDTO updateEvent(EventDTO eventDTO) {
+        Event event= eventRepository.findById(eventDTO.getId()).orElseThrow(()->  new EventNotFoundException("Event not found with id: "));
+        Event eventEntityBeUpdated = EventDTO.mapEventDtoToEvent(eventDTO);
+        User user= userRepository.findById(eventDTO.getUserId()).orElseThrow();
+        eventEntityBeUpdated.setUser(user);
+        Event eventResponseEntity = eventRepository.save(eventEntityBeUpdated);
+        return EventDTO.mapEventToEventDto(eventResponseEntity);
```

Comment:
This method looks too tight. We can use at least one empty row before the `return` step.